docs/DeveloperGuide.md

```diff
@@ -404,6 +404,24 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 --------------------------------------------------------------------------------------------------------------------
 
+## **Appendix: Planned Enhancement**
+
+1. Currently, if any component (name, email, etc.) is too long, it will be hidden on the screen. We plan to implement
+   appropriate mechanism to show the full component.
+2. Currently, names like "`John Doe`" and "`John  Doe`" (two spaces in the middle) are considered different. We plan
+   to have more sophisticated duplication detection mechanism.
+3. Currently, `undo` only supports undoing one command. We plan to supports undoing multiple commands, as well as
+   implementing a `redo` command.
+4. Currently, a professor must have an address attached. This is reasonable in most cases; however, some visiting
+   instructor may not have a specific office address in NUS. Therefore, we plan to allow omitting the field if the user desires.
```

Comment:
Is this not quickly fixable?

Reply:
Part of the reason is it's @LargeCrowd 's feature so if I touch it then RepoSense...

Reply:
What's the suggestion? To just remove the mandatory address for Professors entirely?

Reply:
Best to revert back to optional address for all contacts regardless of roles. But ye as E0735 said, your past efforts will be for naught